src/js/_store/modules/category.js

```diff
@@ -83,18 +90,60 @@ export default {
         });
       });
     },
-    getAllCategories({ commit, rootGetters }) {
+    deleteCategory({ commit, rootGetters, state }) {
+      return new Promise(resolve => {
+        commit('clearMessage');
+        const deletedataid = parseInt(state.deleteCategory.id.categoryId, 10);
+        axios(rootGetters['auth/token'])({
+          method: 'DELETE',
+          url: `/category/${deletedataid}`,
+        }).then(() => {
+          commit('doneMessage', 'カテゴリーの削除が完了しました。');
+          resolve();
+        }).catch(err => {
+          commit('failRequest', { message: err.message });
+        });
+      });
+    },
+    clearMessage({ commit }) {
+      commit('clearMessage');
+    },
+    confirmDeleteCategory({ commit }, payload) {
+      commit('confirmDeleteCategory', { payload });
+    },
+    getUpdateCategory({ commit, rootGetters }, id) {
```

Comment:
[nits]

シンプルに以下で良いかと思います！

```suggestion
    getCategory({ commit, rootGetters }, id) {
```

更新機能以外にも詳細の取得をする時はあるかもしれないので！

Reply:
33a9337cb3ead95a488bfff9363e69675aee88bd

category.jsファイルのgetUpdateCategoryをgetCategoryに修正しました！